sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_exporter.py

```diff
@@ -68,6 +68,9 @@ def _span_to_envelope(self, span: Span) -> TelemetryItem:
         envelope.instrumentation_key = self._instrumentation_key
         return envelope
 
+    @classmethod
+    def from_connection_string(cls, conn_str: str, **kwargs: Any) -> "AzureMonitorTraceExporter":
```

Comment:
missing doc strings

Reply:
also we must remove documenting connection_string in the constructor

Reply:
we must make sure we docuement explicitly in doc strings to use this when providing a conn_str, else use default constructor (for example, when env variable is set)